R/geo.R

```diff
@@ -166,5 +168,86 @@ ward_enr_aggs <- function(df) {
   agg_enr_column_order(df)
 }
 
+#' Aggregates assessment data by ward
+#'
+#'
+#' @param list_of_dfs output of \code{fetch_all_parcc} or \code{fetch_parcc}
+#'
+#' @return A data frame of ward aggregations
+#'
+#'
+#' @export
+ward_parcc_aggs <- function(list_of_dfs) {
+   
+   df <- list_of_dfs %>%
+      bind_rows() %>% # convert to df
+      enrich_school_latlong() %>%
+      enrich_school_city_ward()
+   
+   df <- df %>%
+      filter(!is.na(ward) #,
+             #subgroup == "special_education",
+             #grade == 3,
+             #test_name == "ela",
+             #testing_year == 2018,
+             #!is.na(number_of_valid_scale_scores)
+             ) %>%
+      group_by(
+         testing_year,
+         assess_name, test_name,
+         county_id, county_name,
+         district_id, district_name,
+         ward,
+         grade,
+         subgroup, subgroup_type
+      ) %>%
+      summarize(
```

Comment:
nicely done here! there was some existing code in `parcc_calcs` that attemps to do this parcc aggregation - I rewrote this to crib off of that code.  seems to be working but we should probably add more test coverage?